src/third-version.ts

```diff
@@ -391,11 +393,18 @@ function convertParameter(parameter: any): any {
     }
   }
 
+  if(parameter.schema?.$ref) {
+    console.warn('Could not convert parameter object because the `.schema` property was a reference.\nThis have to be changed manually if you want any of the properties included, it will be converted to a default parameter. The reference was ' + parameter.schema?.$ref);
```

Comment:
```suggestion
    console.warn('Could not convert parameter object because the `.schema` property was a reference.\nThis have to be changed manually if you want any of the properties included. For now your parameter is an empty object after conversion. The reference was ' + parameter.schema?.$ref);
```

still not sure why, if we know that pattern is regression, why we cannot have similar message about pattern, that we were not able to covert and parameter is empty object now?

Reply:
This check is only for references, there is a check further down the code that tells you this regression if you use properties not recognized anymore

Reply:
Ahh, okay better error message 👍